spec/rubocop/target_finder_spec.rb

```diff
@@ -11,6 +11,67 @@
   let(:options) { { force_exclusion: force_exclusion, debug: debug } }
   let(:force_exclusion) { false }
   let(:debug) { false }
+  let(:ruby_extensions) do
+    %w[.rb
+       .arb
+       .axlsx
+       .builder
+       .fcgi
+       .gemfile
+       .gemspec
+       .god
+       .jb
+       .jbuilder
+       .mspec
+       .opal
+       .pluginspec
+       .podspec
+       .rabl
+       .rake
+       .rbuild
+       .rbw
+       .rbx
+       .ru
+       .ruby
+       .spec
+       .thor
+       .watchr].freeze
+  end
```

Comment:
Same here.
I'd go with a regular variable. Just to blow people's minds and remind that RSpec is not a separate language, but just Ruby.

Reply:
Well said. :slightly_smiling_face: